kinetic/distribution.yaml

```diff
@@ -2464,6 +2464,23 @@ repositories:
       url: https://github.com/tork-a/dynpick_driver.git
       version: master
     status: maintained
+  e100_base-relase:
```

Comment:
I'm guessing this should be `e100_base-release`, but honestly, we don't need to even put `-release` in here at all.  I'd suggest just `e100_base`.